### PR TITLE
Add implementation for grallochandler Map, Unmap functions

### PR DESCRIPTION
**Pull request by JeevakaPrabu**
Change-Id: Ic2377b02413f7536784cd2560e16bedfc4426992
Tracked-On: None
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>

### CODE REVIEW

**Comment by JeevakaPrabu** [comment]
@kalyankondapally @harishkrupo

**Comment by kalyankondapally** [comment]
@js0701 @harishkrupo @yugang @Kishore409 please review

File: os/linux/gbmbufferhandler.cpp [hunk 1]
```diff
@@ -239,9 +239,9 @@ void *GbmBufferHandler::Map(HWCNativeHandle handle, uint32_t x, uint32_t y,
 #endif
 }
 
-void GbmBufferHandler::UnMap(HWCNativeHandle handle, void *map_data) {
+int32_t GbmBufferHandler::UnMap(HWCNativeHandle handle, void *map_data) {
   if (!handle->bo)
-    return;
+    return -1;
 
   return gbm_bo_unmap(handle->bo, map_data);
```
Comment (harishkrupo) [inline os/linux/gbmbufferhandler.cpp]: gbm_bo_unmap returns void. [Check here](https://github.com/intel/minigbm/blob/master/gbm.h#L343)

Returning void in a function returning int32_t is not allowed. We just return the success code after calling gbm_bo_unmap.

**Comment by kalyankondapally** [comment]
@js0701 @harishkrupo @yugang @Kishore409 please review

**Comment by harishkrupo** [comment]
LGTM

**Comment by kalyankondapally** [comment]
merged thanks

**Pull request closed without merging**